src/hCassandra_test.py

```diff
@@ -0,0 +1,183 @@
+__author__ = 'annyz'
+
+from sys import path
+# Append 'hydra' directory to Python path
+path.append("hydra/src/main/python")
+
+import sys
+import logging
+import math
+import ast
+
+from datetime import datetime, timedelta
+from optparse import OptionParser
+from pprint import pformat  # NOQA
+from hydra.lib import util
+from hydra.lib.h_analyser import HAnalyser
+from hydra.lib.hydrabase import HydraBase
+
+try:
+    # Python 2.x
+    from ConfigParser import ConfigParser
+except ImportError:
+    # Python 3.x
+    from configparser import ConfigParser
+
+l = util.createlogger('hCassandra', logging.DEBUG)
+
+
+class RunTestCassandra(HydraBase):
+    def __init__(self, options, runtest=True, mock=False):
+        self.options = options
+        self.config = ConfigParser()
+        HydraBase.__init__(self, 'CassandraStressTest', self.options, self.config, startappserver=runtest, mock=mock,
+                           app_dirs=['src', 'hydra'])
+        self.stress_client = '/stress-client'
+        self.add_appid(self.stress_client)
+        if runtest:
+            self.run_test()
+            self.stop_appserver()
+
+    def rerun_test(self, options):
+        self.options = options
+        self.reset_all_app_stats(self.stress_client)
+        # Signal message sending
+        l.info("Sending signal to Cassandra Stress client to start sending all messages..")
+        # Force start-time for ALL clients +10 seconds from current time
+        start_time = datetime.now() + timedelta(seconds=10)
+        l.debug("Current Time: %s, Start Time: %s" % (datetime.now(), start_time))
+        task_list = self.all_task_ids[self.stress_client]
+        ha_list = []
+        for task_id in task_list:
```

Comment:
this can be probably a func in base to just start_test for all tasks, like the ping.